.github/scripts/build-gcc.sh

```diff
@@ -12,15 +12,21 @@ if [ $RUN_CONFIG = 1 ] || [ ! -f "$GCC_BUILD_PATH/Makefile" ] ; then
 
     rm -rf $GCC_BUILD_PATH/*
 
-    # REMOVED --libexecdir=/opt/lib
-    # REMOVED --with-{gmp,mpfr,mpc,isl}=/usr
-    $SOURCE_PATH/$GCC_VERSION/configure \
+    case $TARGET in
+        *mingw*)
+            HOST_OPTION=" \
+                --enable-threads=win32 \
+                --disable-win32-registry"
+            ;;
+    esac
+
+    CFLAGS=$CFLAGS $SOURCE_PATH/$GCC_VERSION/configure \
```

Comment:
Is the `CFLAGS=$CFLAGS` still needed? It does not seem to be set anywhere around...

Reply:
Removed